src/dags/calipso_task_queuer.py

```diff
@@ -0,0 +1,52 @@
+from datetime import timedelta
+
+from airflow import DAG
+from airflow.utils import timezone
+from airflow.utils.dates import days_ago
+
+from ergo.config import Config
+from ergo.operators.sqs.sqs_task_pusher import SqsTaskPusherOperator
+from ergo.sensors.task_requests_batcher import TaskRequestBatchSensor
+
+XCOM_REQUEST_TASK_KEY = "request.tasks"
+XCOM_REQUEST_SQS_QUEUE_URL = "request.sqs_url"
+TASK_ID_REQUEST_SENSOR = "calipso_collect_requests"
+
+default_args = {
+    'owner': 'airflow',
+    'depends_on_past': False,
+    'retries': 2,
+    'retry_delay': timedelta(minutes=1),
+    'start_date': days_ago(1),
+}
+
+max_requests = Config.max_requests
+
+max_concurrent_runs = Config.max_runs_dag_task_queuer
```

Comment:
@joelkbiju12 create seperate `max_runs_dag_task_queuer` for each type of queuer.